charts/stable/homebridge/templates/configmap.yaml

```
@@ -1,21 +1,9 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "homebridge.fullname" . }}
+  name: {{ template "common.names.fullname" . }}-config
   labels:
-{{ include "homebridge.labels" . | indent 4 }}
+    {{- include "common.labels" . | nindent 4 }}
 data:
   startup.sh: |
-    #!/bin/sh
-    
-    # Docker Homebridge startup.sh overriden by Helm Chart
-    
-    # Plugins
-    {{- range .Values.config.plugins }}
-    npm install {{ . }}
-    {{- end }}
-
-    # Packages
-    {{- range .Values.config.additionalPackages }}
-    apk add --no-cache {{ . }}
-    {{- end }}
+{{ .Values.config | indent 4 }}
```

```suggestion
    {{- .Values.config | nindent 4 }}
```
Indenting like this would make it a bit easier to read